frontend/src/components/ui/chat/ChatBlock.js

```diff
@@ -0,0 +1,41 @@
+import { useState } from "react"
```

Comment:
Maybe add semicolon add the end here haha

Reply:
OOO good eye, thanks!